apps/web/src/pages/ApplicantDashboardPage/components/ReviewRecruitmentProcessPreviewList.tsx

```diff
@@ -0,0 +1,575 @@
+                  })}
+                </Link>
+              </Dialog.Footer>
+            )}
+          </div>
+        </Dialog.Body>
+      </Dialog.Content>
+    </Dialog.Root>
+  );
+};
+
+interface ReviewRecruitmentProcessPreviewListProps {
+  recruitmentProcessesQuery: FragmentType<
+    typeof ReviewRecruitmentProcessPreviewList_Fragment
+  >[];
+}
+
+const ReviewRecruitmentProcessPreviewList = ({
+  recruitmentProcessesQuery,
+}: ReviewRecruitmentProcessPreviewListProps) => {
+  const intl = useIntl();
+
+  const recruitmentProcesses = getFragment(
+    ReviewRecruitmentProcessPreviewList_Fragment,
+    recruitmentProcessesQuery,
+  ).filter(
+    (recruitmentProcess) =>
+      recruitmentProcess.finalDecisionAt &&
+      isQualifiedFinalDecision(recruitmentProcess.finalDecision?.value),
+  ); // filter for qualified recruitment processes
+
+  return (
+    <>
+      {recruitmentProcesses.length ? (
+        <PreviewList.Root>
+          {recruitmentProcesses.map((recruitmentProcess) => {
+            const { id, pool, finalDecisionAt, removedAt } = recruitmentProcess;
+
+            const status = getQualifiedRecruitmentStatusChip(
+              recruitmentProcess.suspendedAt,
+              recruitmentProcess.placedAt,
+              intl,
+            );
+
+            const applicationMetadata: PreviewMetaData[] = [
+              {
+                key: "status",
+                type: "chip",
+                color: status.color,
+                children: status.label,
+              },
+              {
+                key: "classification",
+                type: "text",
+                children: pool?.classification
+                  ? getClassificationName(pool?.classification, intl)
+                  : intl.formatMessage(commonMessages.notFound),
+              },
+              {
+                key: "date",
+                type: "text",
+                children: (
+                  <RecruitmentDate
+                    finalDecisionAt={finalDecisionAt}
+                    removedAt={removedAt}
+                    status={status.value}
+                  />
+                ),
+              },
+            ];
+
+            return (
+              <PreviewList.Item
+                key={id}
+                title={
+                  pool.name?.localized ??
+                  intl.formatMessage(commonMessages.notFound)
+                }
+                metaData={applicationMetadata}
+                action={
+                  <ReviewRecruitmentProcessDialog
+                    recruitmentProcessQuery={recruitmentProcess}
+                  />
+                }
+                headingAs="h4"
+              />
+            );
+          })}
+        </PreviewList.Root>
+      ) : (
+        <Well data-h2-text-align="base(center)">
+          <p data-h2-font-weight="base(bold)">
+            {intl.formatMessage({
+              defaultMessage: "You don't have any applications at the moment.",
```

Comment:
It looks like the null state copy for this is different in the designs.
![image](https://github.com/user-attachments/assets/4a03b1bb-780a-4bd1-8c21-5e620a04e8e9)

Reply:
Fixed in [91a34bc](https://github.com/GCTC-NTGC/gc-digital-talent/pull/12366/commits/91a34bcc8e0e630bf731050ab7e818c47aba6f1e)